### PR TITLE
Rewrite 16 holograph treasury tests

### PR DESCRIPTION
**Pull request by mmvazzano**
## Describe Changes

- Migration of the test 16_holograph_treasury_tests to foundry. 
## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Code styles have been enforced
- [x] All Foundry tests are passing

### CODE REVIEW

File: test/foundry/deploy/16_holograph_treasury_tests.t.sol [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+  /*                                 setBridge()                                */
+  /* -------------------------------------------------------------------------- */
+
+  /**
+   * @notice Tests that an admin can alter the bridge slot
+   * @dev This test verifies that an admin can successfully alter the bridge slot by
+   * calling the `setBridge` function.
+   * Refers to the hardhat test with the description 'should allow admin to alter _bridgeSlot'
+   */
+  function testAtllowAdminToAlterBridgeSlot() public {
```
Comment (djaciel) [inline test/foundry/deploy/16_holograph_treasury_tests.t.sol]: This is a typo

File: test/foundry/deploy/16_holograph_treasury_tests.t.sol [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+  /*                               setHolograph()                               */
+  /* -------------------------------------------------------------------------- */
+
+  /**
+   * @notice Tests that an admin can alter the holograph slot
+   * @dev This test verifies that an admin can successfully alter the holograph slot by
+   * calling the `setHolograph` function.
+   * Refers to the hardhat test with the description 'should allow admin to alter _holographSlot'
+   */
+  function testAtllowAdminToAlterHolographSlot() public {
```
Comment (djaciel) [inline test/foundry/deploy/16_holograph_treasury_tests.t.sol]: This is a typo

File: test/foundry/deploy/16_holograph_treasury_tests.t.sol [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+  /*                               setOperator()                               */
+  /* -------------------------------------------------------------------------- */
+
+  /**
+   * @notice Tests that an admin can alter the operator slot
+   * @dev This test verifies that an admin can successfully alter the operator slot by
+   * calling the `getOperator` function.
+   * Refers to the hardhat test with the description 'should allow admin to alter _operatorSlot'
+   */
+  function testAtllowAdminToAlterOperatorSlot() public {
```
Comment (djaciel) [inline test/foundry/deploy/16_holograph_treasury_tests.t.sol]: this is a typo

**Review by djaciel — APPROVED** [review]
I left just a comment

File: test/foundry/deploy/16_holograph_treasury_tests.t.sol (hunk 1 above)
Comment (mmvazzano) [reply test/foundry/deploy/16_holograph_treasury_tests.t.sol]: Typo fixed

File: test/foundry/deploy/16_holograph_treasury_tests.t.sol (hunk 3 above)
Comment (mmvazzano) [reply test/foundry/deploy/16_holograph_treasury_tests.t.sol]: Typo fixed

File: test/foundry/deploy/16_holograph_treasury_tests.t.sol (hunk 2 above)
Comment (mmvazzano) [reply test/foundry/deploy/16_holograph_treasury_tests.t.sol]: Typo fixed

**Review by sogoiii — APPROVED** [review]
👍

**Review by alexanderattar — APPROVED** [review]

**Pull request merged**